stylesheet.css

```diff
@@ -1,282 +1,61 @@
-    position: absolute;
-    top: 0;
-  }
-  
-  /* The dots/bullets/indicators */
-  .dot {
-    cursor: pointer;
-    height: 15px;
-    width: 15px;
-    margin: 0 2px;
-    background-color: #bbb;
-    border-radius: 50%;
-    display: inline-block;
-    transition: background-color 0.6s ease;
-  }
-  
-  .active,
-  .dot:hover {
-    background-color: #717171;
-  }
-  
-  /* Fading animation */
-  .fade {
-    -webkit-animation-name: fade;
-    animation-name: fade;
-    -webkit-animation-duration: 1.5s;
-    animation-duration: 1.5s;
-  }
-  
-  @-webkit-keyframes fade {
-    from {
-      opacity: 0.4;
-    }
-  
-    to {
-      opacity: 1;
-    }
-  }
-  
-  @keyframes fade {
-    from {
-      opacity: 0.4;
-    }
-  
-    to {
-      opacity: 1;
-    }
-  }
-  
-  .loginbg
-  {
-    background-color: #5c5cdd;
-  }
-  
-  .adminbg
-  {
-    background-color:pink;
-  }
-  .servicebg
-  {
-    background-color: #5c5cdd;
-  }
-  
-  .gallerybg
-  {
-    background-image: url("images/sleepy_kid.jpg");
-    background-size: cover;
-  }
-  
-  .contact Usbg
-  {
-    background-color: #5c5cdd;
-  }
-  
-  .girl_woods
-  {
-    width: 20vw;
-    height: 20vh;
-  }
-  
-  .finding_us
-  {
-    width: 20vw;
-    height: 20vh;
-  }
-  
-  .phone_photo
-  {
-    width: 20vw;
-    height: 20vh;
-  }
-  
-  .vintage_camera
-  {
-    width: 20vw;
-    height: 20vh;
-  }
-  
-  form {
-    display: flex;
-    flex-direction: column;
-  }
-  
-  forn > span {
-    color: red;
-  }
-  
-  form > * {
-    padding-top: 15px;
-    padding-bottom: 15px;
-  }
-  
-  form > input {
-    padding: 15px;
-    border: none;
-    background-color: rgb(235, 235, 235);
-  }
-  
-  form > button {
-    padding: 15px;
-  }
-  
-  form > textarea {
-    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
-  }
-  
-  /*# sourceMappingURL=style.css.map */
\ No newline at end of file
+  background-color: #181818;
+  width: 100%;
+  height: 100vh;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  overflow-x: hidden;
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+  color:white;
+}
+
+.login page {
+  width: 400px;
+  align-items: center;
+}
+
+body header {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0px 5px 15px 0px rgba(0, 0, 0, 0.43);
+  border-bottom: 2px solid gray;
+}
+
+body header .active {
+  background-color: black;
+  color: white;
+}
+
+body header ul {
+  list-style: none;
+}
+
+body header ul li {
+  display: inline;
+  margin-right: 20px;
+}
+
+body header ul li>a {
+  padding: 15px;
+  background-color: black;
+  border-radius: 5px;
+  transition: all 0.5s ease-in-out;
+  text-decoration: none;
+}
+
+.formcontainer{
+  width: 80%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  box-shadow: 0px 5px 15px 0px rgba(0, 0, 0, 0.43);
+  border-bottom: 2px solid gray;
+  flex-wrap: wrap;
+  flex-direction: column;
+  border: 1px solid white;
+  
+
```

Comment:
You should add some more CSS to add a `margin-bottom: 16px;` to all inputs, so they are separated out.